src/app/app.ts

```diff
@@ -1174,7 +1180,6 @@ function InitializeWaterComponent(): void {
     linechartObj.appendTo('#balance');
     linechartObj.refresh();
 }
-
```

Comment:
Don't remove spaces, line break, unnecessarily

Reply:
Unnecessary removal of spaces has been avoided